.pipelines/singletenancy/cilium-overlay/cilium-overlay-e2e-step-template.yaml

```diff
@@ -42,6 +42,7 @@ steps:
         echo "deploy Cilium ConfigMap"
         kubectl apply -f cilium/configmap.yaml
         kubectl apply -f test/integration/manifests/cilium/cilium${FILE_PATH}-config.yaml
+        kubectl apply -f test/integration/manifests/cilium/hubble/hubble-peer-svc.yaml
```

Comment:
This is overwriting the current cilium-config configMap which is being set on the line above. Did the tests for nightly version of cilium pass? 

Are we only trying to use this configmap (and test hubble) for the nightly pipeline?

Reply:
this actually a service targeting cilium pods, the configmap below is applied below after the Azilium stages have completed

if preferred, @jshr-w, can you move the service installation to the test stage below? I don't have a strong preference as this is a [cilium component](https://github.com/cilium/cilium/blob/main/install/kubernetes/cilium/templates/hubble-relay/service.yaml), albeit not used yet

Reply:
My bad, misread it as I was reviewing. Is there any reasoning for overwriting the configMap further down?